doc/builders/special/runinmkshell.section.md

```diff
@@ -0,0 +1,64 @@
+# pkgs.runInMkShell {#sec-pkgs-runInMkShell}
```

Comment:
Not quite happy about the name.
- `runIn` is a prefix that was also used for `runInLinuxVM`, but this is not a derivation wrapper that runs the wrapped builder as part of its builder.
- `MkShell` would suggest that it's specific to that function, but it's more general.
- `mk` used to be for functions that are "data constructors"; functions that return their arguments in a specific, checked form.

Maybe not the best suggestion, but here's one:

```suggestion
# pkgs.buildShell {#sec-pkgs-buildShell}
```

Reply:
What about `buildShellWrapper`?

Reply:
What about `buildShellWrapper`?